src/compose/images.ts

```diff
@@ -38,7 +38,7 @@ interface FetchProgressEvent {
 }
 
 export interface Image {
```

Comment:
The changes to this are because of `imageForService` returning an "Image" without these properties